peepholelib/models/svd.py

```diff
@@ -8,8 +11,7 @@
 from tensordict import TensorDict
 from tensordict import MemoryMappedTensor as MMT
 
-def c2s(input_shape, weight, bias, stride=(1, 1), padding=(0, 0), dilation=(1,1), device='cpu', verbose=False, warns=True):
-
+def c2s(input_shape, weight, groups, bias, stride=(1, 1), padding=(0, 0), dilation=(1,1), device='cpu', verbose=False, warns=True):
```

Comment:
at this point we should pass the layer and get `weight, bias, stride, padding, dilation, groups` from the layer object.